src/js/utils/index.d.ts

```diff
@@ -50,7 +50,11 @@ export { isObject, deepFreeze, deepMerge, removeUndefined };
  * type SomeComponentProps = PropsOf<typeof SomeComponent>;
  * ```
  */
-export type PropsOf<TComponent> = TComponent extends React.ComponentType<infer P> ? P : never;
+export type PropsOf<TComponent> = TComponent extends React.ComponentType<
+  infer P
+>
+  ? P
+  : never;
```

Comment:
Please revert this change unless you changed the code's meaning.

Reply:
I believe this is because of the linter.

Reply:
Please revert it, my linter doesn't apply it, might be something with your IDEA.